components/cli/vendor/github.com/docker/docker/pkg/archive/archive.go

```diff
@@ -93,6 +93,16 @@ const (
 	OverlayWhiteoutFormat
 )
 
+const (
```

Comment:
Brought in with PR:
* https://github.com/moby/moby/pull/33935 pkg/archive.FileInfoHeader: fill file type bits